recipes/list_items/contact_info/contact_info.vue

```diff
@@ -12,7 +12,8 @@
       >
         <dt-avatar
```

Comment:
I'm a bit confused it looks like it's still showing the old presence style? Shouldn't it be a circle?

![Screenshot 2023-03-14 at 1 52 35 PM](https://user-images.githubusercontent.com/64808812/225133720-8893a1bc-e4f7-4718-8380-d87ec8a02f84.png)

Reply:
It shows circle in my test:
![Screenshot 2023-03-14 at 1 34 59 PM](https://user-images.githubusercontent.com/61763780/225138745-8322f314-6e3b-4b2e-b6c1-af7550c4ffb0.png)
I guess you may need to do some cache clean up.

Reply:
Yeah for some reason the deploy previews have been extremely slow to update lately... it looks fine now.